int32   left_hip_enc_rev
float64  left_hip_pos
float64  left_hip_vel
float64  left_hip_iq

int32   left_knee_enc_rev
float64  left_knee_pos
float64  left_knee_vel
float64  left_knee_iq

int32   left_wheel_enc_rev
float64  left_wheel_pos
float64  left_wheel_vel
float64  left_wheel_iq

int32   right_hip_enc_rev
float64  right_hip_pos
float64  right_hip_vel
float64  right_hip_iq

int32   right_knee_enc_rev
float64  right_knee_pos
float64  right_knee_vel
float64  right_knee_iq

int32   right_wheel_enc_rev
float64  right_wheel_pos
float64  right_wheel_vel
float64  right_wheel_iq